### PR TITLE
[French translation] Minor fixes

### PR DESCRIPTION
**Pull request by SansPseudoFix**

### CODE REVIEW

File: locale/fr/LC_MESSAGES/TaskBar.po [hunk 1]
```diff
@@ -390,7 +390,7 @@ msgid ""
 "The Icon can be Reset within its own Settings."
 msgstr ""
 "Réinitialiser Bouton Plateau avec les paramètres originaux.\n"
-"L'icone peuvent être réinitialisés dans leur propre menu paramètres."
+"L'icône peuvent être réinitialisées dans leur propre menu paramètres."
```
Comment (lcallarec) [inline locale/fr/LC_MESSAGES/TaskBar.po]: Good catch. But, also, there's a plural / singular mismatch : `icône peuvent`. Anyway, the original message show that the whole sentence should be rewritten using the singular form :)

Comment (SansPseudoFix) [inline locale/fr/LC_MESSAGES/TaskBar.po]: I'm a little bit ashamed here. It will be fix soon, thx :)

**Comment by SansPseudoFix** [comment]
Should be okay now.

**Comment by zpydr** [comment]
Hi, I just uploaded a new TaskBar.pot file, since I am planning to release a new version. If you are interested to contribute, could you please contact [narzb](https://github.com/narzb), he translated previous versions. Thank you.

**Pull request closed without merging**